client/my-sites/marketing/tools/feature.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { ReactNode, FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+
+interface MarketingToolFeatureProps {
+	children: ReactNode;
+	description: string;
+	disclaimer?: string;
+	title: string;
+}
+
+const MarketingToolFeature: FunctionComponent< MarketingToolFeatureProps > = ( {
+	children,
+	description,
+	disclaimer,
+	title,
+} ) => {
+	return (
+		<Card className="tools__feature-list-item">
+			<div className="tools__feature-list-item-body">
+				<CardHeading>{ title }</CardHeading>
+				<p>{ description }</p>
+				{ disclaimer && <p className="tools__feature-list-item-disclaimer">{ disclaimer }</p> }
+			</div>
+			<div className="tools__feature-list-item-child-row">{ children }</div>
+		</Card>
+	);
+};
+
+MarketingToolFeature.propTypes = {
+	description: PropTypes.string.isRequired,
+	disclaimer: PropTypes.string,
+	title: PropTypes.string.isRequired,
+};
```

Comment:
Should we keep those prop types since we now have the `MarketingToolFeatureProps` interface?

Reply:
The interface will take care of compile time issues and the PropTypes will be runtime. In this case since this Component is only used by its TypeScript parent we can safely scrap them here.